### PR TITLE
[TASK] Remove "simply" and "just" where not appropriate

### PR DESCRIPTION
**Pull request by brotkrueml**
Inspired by: https://twitter.com/marcelpociot/status/1594706393696985088

Also, some of the usages are just fillers which does not convey meaning.

Releases: main, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/DependencyInjection/Index.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ also talk about how to transition away from it towards the core-wide Symfony sol
 Background and history
 ======================
 
-Obtaining object instances in TYPO3 has always been pretty straight: Just call
+Obtaining object instances in TYPO3 has always been pretty straight: call
```
Comment (linawolf) [inline Documentation/ApiOverview/DependencyInjection/Index.rst:66]:
```suggestion
Obtaining object instances in TYPO3 has always been pretty straight: Call
```

File: Documentation/ApiOverview/Workspaces/Index.rst [hunk 2]
```diff
@@ -69,7 +69,7 @@ The most basic form of a preview is when a live record is selected and
 you lookup a future version of that record belonging to the current
 workspace of the logged in backend user. This is very easy as long as
 a record is selected based on its "uid" or "pid" fields which are not
-subject to versioning; You simply call :code:`sys_page->versionOL()` after
+subject to versioning; You call :code:`sys_page->versionOL()` after
```
Comment (linawolf) [inline Documentation/ApiOverview/Workspaces/Index.rst:72]:
```suggestion
subject to versioning; Call :code:`sys_page->versionOL()` after
```

File: Documentation/ApiOverview/Xclasses/Index.rst [hunk 3]
```diff
@@ -96,7 +96,7 @@ In this example, we declare that the :code:`\TYPO3\CMS\Backend\Controller\NewRec
 will be overridden by the :code:`\T3docs\Examples\Xclass\NewRecordController`
 class, the latter being part of the :t3ext:`examples` extension.
 
-When XCLASSing a class that does not use namespaces, simply use that class' name
+When XCLASSing a class that does not use namespaces, use that class' name
```
Comment (linawolf) [inline Documentation/ApiOverview/Xclasses/Index.rst:99]:
```suggestion
When XCLASSing a class that does not use namespaces, use that class name
```

File: Documentation/ApiOverview/Workspaces/Index.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/ApiOverview/Workspaces/Index.rst:72]: Nope, here not: https://libguides.royalroads.ca/punctuation-capitalization/capsemicolons

Comment (brotkrueml) [reply Documentation/ApiOverview/Workspaces/Index.rst:72]: The whole semicolons in this part do not belong here. Will have a look.

File: Documentation/ApiOverview/Workspaces/Index.rst [hunk 4]
```diff
@@ -69,7 +69,7 @@ The most basic form of a preview is when a live record is selected and
 you lookup a future version of that record belonging to the current
 workspace of the logged in backend user. This is very easy as long as
 a record is selected based on its "uid" or "pid" fields which are not
-subject to versioning; You simply call :code:`sys_page->versionOL()` after
+subject to versioning: call :code:`sys_page->versionOL()` after
```
Comment (brotkrueml) [inline Documentation/ApiOverview/Workspaces/Index.rst:72]: The lowercase "c" in "call" is correct here:

"When a colon introduces a phrase or an incomplete sentence that is meant to add information to the sentence before it, do not capitalize the first word after the colon unless it is a proper noun."
https://www.grammarly.com/blog/capitalization-after-colons/

**Review by linawolf — APPROVED** [review]

**Review by tomwarwick — APPROVED** [review]

**Review by linawolf — APPROVED** [review]

**Pull request merged**